src/typography/Italic/Italic.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { FC, forwardRef, Ref } from 'react';
+export interface ItalicRef {
+  ref?: Ref<HTMLElement>;
+}
+
+export const Italic: FC<React.HTMLAttributes<HTMLElement> & ItalicRef> =
```

Comment:
Let's make these a style/variant/something on the `Text` component.

Reply:
> Let's get rid of the italic/bold/thin and move them into text. 
👍 

> Also is there a way we can make this a little more automated? Seems like we have the same css repeated quite a bit. Maybe something like this:
> 
> ```
> // css file
> @value class-to-compose from "file-where-class-is-defined.css";
> 
> .someclass {
>   composes: class-to-compose;
>   // other styles
> }
> ```

I can do that - my other thought was possibly exporting all of the title/heading components in a single Typography component but that breaks the patterns we have

Reply:
so I tried to use compose but that wasn't working - unsure if I'm missing something 🤔

Reply:
You could import 2 style sheets ?

Reply:
yea we can definitely do that